source/time-series-collections.txt

```diff
@@ -0,0 +1,116 @@
+.. _vsce-time-series:
+
+==============================
+Create Time Series Collections
+==============================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-vsce-preview.rst
+
+You can create 
+:manual:`time series collections </core/timeseries-collections/>` using 
+a MongoDB Playground. 
+
+Prerequisites
+-------------
+
+If you have not done so already, you must complete the following 
+prerequisites before you can create a time series collection with a 
+MongoDB Playground:
+
+- :ref:`Create a connection to a MongoDB deployment 
+  <vsce-connect-task>`.
+- :ref:`Activate the connection to the MongoDB deployment 
+  <vsce-activate-connection>`.
+
+Create a Time Series Collection
+-------------------------------
+
+Once you connect to your deployment using MongoDB for VS code, use the left 
+navigation to: 
+
+1. Expand an active connection and hover over the database where you want 
+   your collection to exist.
+
+#. Click the :icon-fa5:`plus` icon that appears. 
+
+#. A MongoDB playground automatically opens with a template form to create 
+   both regular collections and time series collections.
+
+#. Delete the regular collection form and uncomment the time series form.
+
+#. Fill in the provided fields with details for your time series collection.
+
+#. To run the playground, click the :guilabel:`Play Button` at the top right
+   of the VS code navigation bar.
+
+After running the playground with the time series collection, the left 
+navigation will update the collection icon to identify it is a 
+time series collection.
+   
+.. figure:: /images/vsce-time-series-icon.png
+   :figwidth: 200px
+   :alt: Image showing time series icon in VS Code extension
+
+.. seealso::
+
+   - To learn more about time series collections, see 
+     :manual:`Time Series Collections </core/timeseries-collections/>` 
+
+   - To add documents to the collection, see 
+     :ref:`vsce-create-doc-playground`.  
+
+Example
+-------
+
+To use this example, start with a collection template from your MongoDB 
+Playgrounds.
+
+This example creates a time series collection named ``weather`` in the ``test`` 
+database. 
+
+.. code-block:: javascript
+
+   use('test');
+
+   db.createCollection(
+    "weather",
+    {
+       timeseries: {
+          timeField: "timestamp",
+          granularity: "hours"
+       }
+    }
+  )
+
+In the example:
+
+- The template form provided by MongoDB Playgrounds has been deleted. 
```

Comment:
[suggestion]

I think we can remove this bullet. If you want to retain this information, maybe move it up above to where we introduce the example. We could change the example lead-in to include:

"To run this example, start with a blank MongoDB playground."

Reply:
Done.